tests/components/auth/test_init.py

```diff
@@ -607,17 +631,26 @@ async def test_ws_delete_all_refresh_tokens(
     tokens = result["result"]
 
     await ws_client.send_json(
-        {
-            "id": 6,
-            "type": "auth/delete_all_refresh_tokens",
-        }
+        {"id": 6, "type": "auth/delete_all_refresh_tokens", **delete_token_type}
     )
 
     result = await ws_client.receive_json()
     assert result, result["success"]
+    remaining_tokens = 0
     for token in tokens:
         refresh_token = await hass.auth.async_get_refresh_token(token["id"])
-        assert refresh_token is None
+        if refresh_token is not None:
+            remaining_tokens += 1
```

Comment:
Can you get rid of this block and increase `expected_remaining_tokens` accordingly?

Reply:
I don't follow what you're suggesting. 

We're calling get_refresh_token 4 times in a loop. Each time we get a valid token, increment the counter by 1. And finally compare that against the expected count.

Reply:
Shouldn't you know what the value of `expected_remaining_tokens` is ahead of time? Can you change it to include the increments and get rid of the if block?

Reply:
> Shouldn't you know what the value of expected_remaining_tokens is ahead of time?

Yes I know the expected value, it's fixed as a parameter of the test. 
I'm not incrementing the expected value.
I'm incrementing a counter that tells me how many valid tokens still existed after calling the delete api.

Reply:
Right. Sorry I'm looking on my phone as I was trying to get it in before beta.  I'll wait till I get back to my desk for more feedback.